### PR TITLE
1.0.0 rc.2

### PR DESCRIPTION
**Pull request by helfer**
Let's see what breaks :boom:

### CODE REVIEW

**Comment by helfer** [comment]
Too bad I can't review my own PR 😞  Good thing I can still merge it though 😄

**Comment by stubailo** [comment]
Maybe you should make the tests pass :P

**Comment by helfer** [comment]
Yeah, maybe next time I'll run those before pushing. Tends to look a bit cleaner 😬

**Comment by helfer** [comment]
@stubailo @calebmer since I had to change a few more things than just the Changelog and the version, a quick review by one of you would be great!

File: src/graphql.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ import ApolloClient, {
   ApolloStore,
   ApolloQueryResult,
   ApolloError,
+  // FetchPolicy,
```
Comment (stubailo) [inline src/graphql.tsx]: What's up with this

Comment (helfer) [inline src/graphql.tsx]: Unfortunately it's not exported by Apollo Client yet. "Someone" forgot to export it. We'll add it in the next release.

**Comment by helfer** [comment]
Alright, this is it!

**Pull request merged**